aggregates/src/commonMain/kotlin/org/dhis2/mobile/aggregates/ui/DataSetTableScreen.kt

```diff
@@ -400,6 +445,40 @@ private fun DataSetTableContent(
     }
 }
 
+@Composable
```

Comment:
What do you think about extracting this component to a specific class. In my branch I created a `component` package inside `ui` to store the custom composables for this use case.

![image](https://github.com/user-attachments/assets/e29b85c2-0d95-4cfe-a008-8cc70927cfa2)